fluxion/test_vector.py

```diff
@@ -22,11 +22,11 @@ class TestVector:
     '''Test vector'''
 
     # pylint: disable=redefined-builtin
-    def __init__(self, name: str, source: str, source_hash: str, input: str,
+    def __init__(self, name: str, source: str, source_checksum: str, input: str,
```

Comment:
I think this can be a good moment to rename input to something else as input is a builtin

Reply:
`input_file`? just `file`?